tests/streamlit_ui_tests/streamlit.spec.ts

```diff
@@ -1,9 +1,302 @@
+    await clickButtonAndAwaitResponse(page, mito, 'Format all of the selected columns as percentage. This only changes the display of the data, and does not effect the underlying dataframe.')
+    await expect(mito.getByText('100%')).toBeVisible();
+    await clickButtonAndAwaitResponse(page, mito, 'Increase the number of decimal places that are displayed in the selected number columns.')
+    await expect(mito.getByText('100.0%')).toBeVisible();
+    await clickButtonAndAwaitResponse(page, mito, 'Decrease the number of decimal places that are displayed in the selected number columns.')
+    await expect(mito.getByText('100%')).toBeVisible();
+    await mito.getByText('100%').click();
+  });
+
+  test.skip('Conditional Formatting', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    
+    await clickButtonAndAwaitResponse(page, mito, 'Conditional Formatting')
+    
+    await checkOpenTaskpane(mito, 'Conditional Formatting');
+    
+    // This test is currently broken, for some reason this button won't click
+    await mito.getByRole('button', { name: 'Add Conditional Formatting Rule' }).click();
+    await mito.locator('div').filter({ hasText: /^Is not empty Applied to 0 columns\.$/ }).first().click();
+    await mito.getByRole('checkbox').nth(1).check(); // Check the first column
+    await mito.getByRole('textbox').nth(2).fill('#b32929'); // Set the color
+
+    // Check that the .mito-grid-cell containing 1 has 
+    // background-color: rgba(179, 41, 41, 0.4)
+    await expect(mito.locator('.mito-grid-cell').filter({ hasText: /^1$/ }).first()).toHaveAttribute('style', /background-color: rgba\(179, 41, 41, 0.4\)/);
+    // Check that the .mito-grid-cell containing 2 does not 
+    // background-color: rgba(179, 41, 41, 0.4)
+    await expect(mito.locator('.mito-grid-cell').filter({ hasText: /^2$/ }).first()).not.toHaveAttribute('style', /background-color: rgba\(179, 41, 41, 0.4\)/);
+  
+    await mito.getByRole('button', { name: 'Import Files' }).click();
+    await mito.getByText('test.csv').click();
+    await mito.getByText('test.csv', { exact: true }).click();
+    await mito.locator('div').filter({ hasText: /^test\.csv$/ }).nth(1).click();
+    await mito.getByRole('button', { name: 'Import test.csv' }).click();
+    await mito.getByRole('button', { name: 'Import test.csv' }).click();
+    await mito.locator('#mito-center-content-container line').nth(1).click();
+    await mito.getByText('Column3').click();
+    await mito.locator('[id="mito-toolbar-button-add\\ column\\ to\\ the\\ right"]').getByRole('button', { name: 'Insert' }).click();
+    await mito.locator('#cell-editor-input').fill('=123');
+    await mito.locator('#cell-editor-input').press('Enter');
+  
+  });
+
+  test.skip('Color Dataframe', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+
+    await clickButtonAndAwaitResponse(page, mito, { name: 'Format', exact: true })
```

Comment:
How come you sometimes use `clickButtonAndAwaitResponse` and other times just use `getByRole.....click()`

Reply:
Added to README.